scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua

```diff
@@ -26,3 +38,26 @@ function onMobDespawn(mob)
     DeterMob(PH, false);
     GetMobByID(PH):setRespawnTime(GetMobRespawnTime(PH));
 end;
+
+-----------------------------------
+-- onAdditionalEffect
+-----------------------------------
+
+function onAdditionalEffect(mob, player)
+    local chance = 25;
+    local resist = applyResistanceAddEffect(mob,player,ELE_WATER,EFFECT_PLAGUE);
+    if (math.random(0,99) >= chance or resist <= 0.5) then
+        return 0,0,0;
+    else
+        local duration = 30;
+        if (mob:getMainLvl() > player:getMainLvl()) then
+            duration = duration + (mob:getMainLvl() - player:getMainLvl())
+        end
+        duration = utils.clamp(duration,1,45);
+        duration = duration * resist;
+        if (not player:hasStatusEffect(EFFECT_PLAGUE)) then
```

Comment:
Shouldn't actually have to do this. If the player already has it `addStatusEffect()` will normally do nothing unless the effect is stronger (a few effects have flags enabling them to behave differently, but I don't think this is one of them).

Guessing you used King Vinegaroon or one of the badly written additional effect weapons as a model based on this and the attempted level correction. There is a lot we don't know in regard to how retail does it when parts of the additional effect is variable. I copypasta'd a lot of these myself. I'll leave it a few days to see if anyone else has any comments on what should happen here and nobody dies I'll merge (silence is acceptance, and I lack time to be as picky on resists checks and crap these days).